demo/python/multiclient-distributed/distr-training.py

```diff
@@ -0,0 +1,86 @@
+import securexgboost as xgb
+import os
+
+DIR = os.path.dirname(os.path.realpath(__file__))
+HOME_DIR = DIR + "/../../../"
+
+train_enc_1 = HOME_DIR + "demo/python/multiclient/train1.enc"
```

Comment:
Are these files checked in? If not, we should generate them using `xgb.encrypt_file`